deepchecks/vision/context.py

```diff
@@ -128,11 +130,14 @@ def __init__(self,
                         logger.warning('validate_prediction() was not implemented in %s dataset, '
                                        'some checks will not run', dataset_type)
 
-        # Set seeds to if possible (we set it after the validation to keep the seed state)
-        if train and random_state:
-            train.set_seed(random_state)
-        if test and random_state:
-            test.set_seed(random_state)
+        # The copy does 2 things: Sample n_samples if parameter exists, and shuffle the data.
+        # we shuffle because the data in VisionData is in fixed order, so if the user wants to run without random_state
```

Comment:
```suggestion
        # we shuffle because the data in VisionData is set to be sampled in a fixed order, so if the user wants to run without random_state
```